components/search/list-renderer/prediction-set.js

```diff
@@ -27,9 +27,17 @@ export default function PredictionSet({ item: predictionSet }) {
           ...new Set(predictionSet.files.map((file) => file.content_type)),
         ].sort()
       : [];
+  const sampleSummaries =
+    predictionSet.samples?.length > 0
+      ? [
+          ...new Set(predictionSet.samples.map((sample) => sample.summary)),
+        ].sort()
```

Comment:
This does case-sensitive sorting, though that might be fine. All the prediction-set summaries I see right now start with a lower-case letter.